### ISSUE TITLE
Mixed German and English Notation of H/B/Bb

### ISSUE THREAD

**Issue by JosuaHaug** [cgnieder/leadsheets#4]
I would prefer to write "H"(de)/"B"(en) always as "H" while writing "B"(de)/"Bb"(en) always as "Bb".
I know it is not theoretically wrong, but for my musician colleges it is more clear to write it that way.
If I write the chords like this:
```latex
\begin{song}{}
\begin{chorus}
^{H}Eitsch and not ^{Bb}Bee
\end{chorus}
\end{song}
```
I get the desired Output, but transposing the song won't work in none of the possible combinations of (german/english input/output)

Is there a way to achieve it with the current version or is there a simple workaround?

**Comment by cgnieder** [comment]
Hmm, there is definitely room for improvement... I'll think of something. Unfortunately there is no easy way with the current version: the transposing mechanism does not recognize `H` at all and simply ignores it. Worse: `B` with German input is still treated as non-German input when transposed:

```latex
\documentclass{article}
\usepackage{leadsheets}

\setleadsheets{
  chordnames/input-notation = german ,
  chordnames/output-notation = german
}

\begin{document}

\setleadsheets{transpose=2}

\begin{song}{key=C}
\begin{chorus}
  ^{H}ignored ^{B}wrong
\end{chorus}
\end{song}

\end{document}
```

**Comment by cgnieder** [comment]
I will very likely drop the `literal` input/ouput notation option and only leave `german` and `english`. This would easy the fix for the wrong transposing a lot.
Additionally I'm thinking of offering options to modify the German notations. This would mean a user could declare that B flat should be output as “Bb” but B as “H”.

**Comment by cgnieder** [comment]
This is fixed/implemented in commit 8ad70c1a4af5daa81ec3f448108baeaa7338b1d7
